src/unitxt/operators.py

```diff
@@ -734,3 +735,60 @@ def process(self, instance: Dict[str, Any], stream_name: str = None) -> Dict[str
             dict_set(instance, field, new_values, use_dpath=True, set_multiple=True)
 
         return instance
+
+
+class StreamRefiner(SingleStreamOperator):
+    max_instances: int = None
```

Comment:
Why is this a class member? should it be a local variable?

Reply:
It is to allow control over the max instances of all the streams instead of having to define max per stream